cmd/imageproxy/main.go

```diff
@@ -87,6 +88,7 @@ func main() {
 
 	p.IncludeReferer = *includeReferer
 	p.FollowRedirects = *followRedirects
+	p.MaxRedirects = uint8(*maxRedirects)
```

Comment:
`254` looks a reasonable max value for me considering that we can also just set `0` to disable the limiting for good.